src/js/edu-benefits/definitions/veteranId.js

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash/fp';
+
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+// TODO: move vaFileNumber out of 1995 and into a shared backend definition
```

Comment:
In other pages/definitions we've been passing in the schema to a function, so that's a possibility, too.